tools/test-proxy/Azure.Sdk.Tools.TestProxy/ConsoleWrapper/ConsoleWrapperImpl.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+using System;
+
+namespace Azure.Sdk.Tools.TestProxy.ConsoleWrapper
+{
+    /// <summary>
+    /// Implementation of IConsoleWrapper that's simply a passthrough to the Console functions.
+    /// </summary>
+    public class ConsoleWrapperImpl : IConsoleWrapper
```

Comment:
```suggestion
    public class ConsoleWrapper : IConsoleWrapper
```

Reply:
As per our conversation I changed the namespace to Azure.Sdk.Tools.TestProxy.Console and the class to ConsoleWrapper.